docs/scripting-ref.rst

```diff
@@ -2230,3 +2230,128 @@ Trigger mouse release event in web page.
 **Async:**: no.
 
 See notes about mouse events in :ref:`splash-mouse-click`.
+
+.. _splash-send-keys:
+
+splash:send_keys
+----------------
+
+Send keyboard events to page context.
+
+**Signature:** ``splash:send_keys(keys)``
+
+**Parameters**
+
+* keys - string representing the keys to be sent as keyboard events.
+
+**Returns:** nil
+
+**Async:**: no.
+
+Key sequences are specified by using a small subset of emacs edmacro syntax:
+
+* whitespace is ignored and only used to separate the different keys
+* characters are literally represented
+* ``RET``, ``SPC``, ``TAB``, ``DEL``, ``ESC`` are reserved and represent
+  special control characters.
+* words within brackets represent function keys, like ``<Return>``, ``<Left>``
+  or ``<Home>``. See `Qt docs`__ for a full list of function keys. ``<Foo>``
+  will try to match ``Qt::Key_Foo``.
```

Comment:
Heh. I'm on fence about that: on one hand, 3-letter shortcuts provide a second way to do the same as `<Shortcuts>`, this is not good. I don't see what're the advantages: `<Space>` is more readable than `SPC`. On another hand, this way the format becomes compatible with the existing format (edmacro), and this is awesome. I'd prefer not to invent a format if we can. But only a small subset of edmacro is implemented.

Is there a reason why 'special function keys' are separated from regular 'function keys' in edmacro format?